job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/DatabasePoller.java

```diff
@@ -118,6 +71,17 @@ public static void pollDatabaseForJobsToRun() {
         }
     }
 
+    private static void sendMessageToQueueMessaging(final Codec codec, final JobTaskData jtd, final WorkerAction workerAction)
+    {
+        try (final QueueServices queueServices= QueueServicesFactory.create(jtd.getTaskPipe(), codec)){
+            LOG.debug("Sending task data to the target queue {} ...", workerAction);
+            queueServices.sendMessage(jtd.getPartitionId(), jtd.getJobId(), workerAction);
+            deleteDependentJob(jtd.getPartitionId(), jtd.getJobId());
+        } catch(final Exception ex) {
+            LOG.warn("Failed to send message about dependent jobs {}", ex.getMessage());
```

Comment:
Don't just log the exception message.

```suggestion
            LOG.warn("Failed to send message about dependent jobs", ex);
```